File: app/redis-client.js
const Redis = require("ioredis");
const redis_client = new Redis({
    ...(process.env.DOCKER ? { host: process.env.REDIS_HOST } : {}),
    port: process.env.REDIS_PORT,
    username: process.env.REDIS_USERNAME,
    password: process.env.REDIS_PASSWORD
});

console.error(process.env.REDIS_USERNAME)

redis_client.on("error", function (error) {
    console.error(error);
});

module.exports.redis_client = redis_client;

File: redis/scripts/redis_client.py
import os
import sys

from dotenv import load_dotenv

import redis

project_dir = os.path.split(os.path.realpath(sys.argv[0]))[0]
os.chdir(project_dir)

load_dotenv('../env/redis.env')

redis_client = redis.Redis(
    host='localhost',
    port=os.getenv('REDIS_PORT'),
    username=os.getenv('REDIS_USERNAME'),
    password=os.getenv('REDIS_PASSWORD'))